developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -7,37 +7,61 @@ import BlockQuote from "@site/src/components/BlockQuote";
 
 # Run a Local Testnet
 
-You can run a local testnet of the Aptos Blockchain. This network runs independently of the Aptos ecosystem and is for testing and development purposes.
+You can run a local testnet of the Aptos Blockchain. This local testnet will not be connected to the Aptos devnet. It will run on your local machine, independently of the Aptos ecosystem. You can use this local testnet for testing and development purposes.
 
-<BlockQuote type="info">
-Note: your local testnet will not be connected to the Aptos devnet. It will run on your local machine and be contained there.
-</BlockQuote>
+You can run a local testnet in two ways:
 
-## Getting Started
+1. Using the Aptos-core source code. This approach is useful for testing modifications to the Aptos-core codebase or to the Aptos Framework.
 
-You can run a local testnet in two ways either using the Aptos-core source code or using Docker:
-1. The Aptos-core source code is useful for testing modifications to the Aptos-core codebase or the Aptos Framework.
-2. Docker is particularly useful for building services on top of the Aptos Blockchain or the Aptos Framework, as there is no build overhead and the ledger persists across network restarts (by default).
+2. Using Docker. This is particularly useful for building services on top of the Aptos Blockchain or the Aptos Framework, as there is no build overhead and the ledger persists across network restarts (by default).
 
-We describe each method below.
+The rest of this document describes:
 
-### Using the Aptos-core source code
+- How to start your local testnet using both the methods, and
+- How to start a Faucet service and attach it to your testnet.
 
-1. Clone the Aptos-core repository from GitHub and prepare your developer environment by running the following commands:
+## Using the Aptos-core source code
+
+1. Clone the Aptos-core repository from GitHub.
+
+  ```
+  git clone https://github.com/aptos-labs/aptos-core.git
+  ```
+
```

Comment:
I used the clone of my fork. Should we recommend fork plus clone, instead of cloning Aptos-core directly?

Reply:
yes, that's ideal.

Reply:
Done.